src/model/ApplicantDetails.ts

```diff
@@ -0,0 +1,13 @@
+import { Nationality } from "./Nationality";
+import { Address } from "./Address";
+
+export interface ApplicantDetails {
+  firstName?: string;
```

Comment:
You dont need to create it here... I thought we were using the Model from api-sdk-node